lib/routes/chikubi/category.ts

```diff
@@ -0,0 +1,49 @@
+import { Route, Data } from '@/types';
+import { processItems } from './utils';
+import Parser from 'rss-parser';
+
+export const route: Route = {
+    path: '/category/:keyword',
+    categories: ['multimedia'],
+    example: '/chikubi/category/nipple-lesbian',
+    parameters: { keyword: 'Keyword' },
+    features: {
+        requireConfig: false,
+        requirePuppeteer: false,
+        antiCrawler: false,
+        supportBT: false,
+        supportPodcast: false,
+        supportScihub: false,
+    },
+    name: 'Category',
+    maintainers: ['SnowAgar25'],
+    handler,
+    radar: [
+        {
+            title: 'Category',
+            source: ['chikubi.jp/category/:keyword'],
+            target: '/category/:keyword',
+        },
+    ],
+};
+
+async function handler(ctx): Promise<Data> {
+    const { keyword } = ctx.req.param();
+    const baseUrl = 'https://chikubi.jp';
+    const url = `/category/${keyword}`;
+
+    const feed = await new Parser().parseURL(`${baseUrl}${url}/feed`);
+
+    const list = feed.items.map((item) => ({
+        title: item.title,
+        link: item.link,
+    }));
+
+    const items = await processItems(list);
```

Comment:
Please use [WordPress REST API](https://developer.wordpress.org/rest-api/) instead.
The API endpoints are available at `/wp-json/wp/v2`.

For categories, you can find the category id first using `/wp-json/wp/v2/categories` with the search parameter `slug`. The id for `nipple-lesbian` is `9`. Then you can list recent posts with the required categories using `/wp-json/wp/v2/posts` with the search parameter `categories`, which will be `9` for `nipple-lesbian`. If you want plain categories and tags for each post in the posts API, you can add the `_embed` search parameter to `/wp-json/wp/v2/posts`

To adjust the no. of articles returned to the posts API, you can pass the `per_page` search parameter to `/wp-json/wp/v2/posts`.